woocommerce/wc-admin
# Update "Deactivate old WCA version" note to be informational

Partially addresses #4502 by updating any existing "Deactivate old WC Admin version" notes to be info notes, per #4629.

It does this via an update function that executes a SQL update, modifying the deactivate plugin note.

### Detailed test instructions:

- `update wp_options set option_value = '1.4.0' where option_name = 'woocommerce_admin_version'` - this stops the update script from running too early
- `delete from wp_wc_admin_notes` - this gets us to a clean starting environment with no notes
- Refresh the page and make sure that the deactivate plugin note is shown in the inbox
- `update wp_wc_admin_notes set type = 'update' where name = 'wc-admin-deactivate-plugin'` - this sets the type to `update` which what most databases that contain this note will have, since the note type has only been changed to `info` in the last few days
- Refresh and observe that the deactivate plugin note should be shown in the store notices card
- `update wp_options set option_value = '1.2.0' where option_name = 'woocommerce_admin_version'` - this ensures that updates in `Install` are executed on next refresh
- `delete from wp_actionscheduler_actions where args = '["wc_admin_update_140_change_deactivate_plugin_note_type"]'` - this makes sure that the update script will execute when we next refresh
- Refresh the page to trigger the database update
- Observe that the deactivate plugin note should now be in the list of admin notices
